templates/docs/examples/utilities/vertical-spacing-common.html

```diff
@@ -0,0 +1,19 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Vertical common spacing{% endblock %}
```

Comment:
On a second thought "common" sounds weird in here. It doesn't add much.

I think it would be best to call this one simply "Vertical spacing" (I guess the same for file name).
And the old example we could rename to "Vertical spacing by unit" (`vertical-spacing-unit.html` for the file).
When renaming files the links for embedded examples will need to be updated as well.